includes/cart/class-edd-cart.php

```diff
@@ -358,7 +358,8 @@ public function get_quantity() {
 	 * @return boolean
 	 */
 	public function is_empty() {
-		return 0 === sizeof( $this->contents );
+		$contents = array( $this->contents );
```

Comment:
I don't believe this code will do what it's intended to do. 

In EDD 3.0, we used:
```
return 0 === count( (array) $this->get_contents() )
```

This does a few different things:
* Casts to `array` rather than accidentally nesting it inside another array
`array( (array) $thing );
* Avoids the cost of aliasing `sizeof()` to `count()`
* Reads a bit more clearly to our EDD 3.0 eyes

cc @sunnyratilal for confirmation

Reply:
@jjj, that makes sense, I can update it if @sunnyratilal confirms